libs/net/dal/Configuration/SeriesConfiguration.cs

```diff
@@ -15,6 +15,12 @@ public override void Configure(EntityTypeBuilder<Series> builder)
 
         builder.HasOne(m => m.Source).WithMany(m => m.Series).HasForeignKey(m => m.SourceId).OnDelete(DeleteBehavior.Cascade);
 
+        // CREATE UNIQUE INDEX "IX_series_name" ON public."series"
```

Comment:
A reminder that the index must be created in a separate script.  Entity Framework doesn't support postgres unique indexing with null values.